src/lib/core/cxx_abi.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2010-2022, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#pragma once
+
+#include "trivia/config.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif /* __cplusplus */
+#ifdef ENABLE_BACKTRACE
+/*
+ * Wrapper around `abi::__cxa_demangle` from the C++ ABI, for details see:
+ * https://gcc.gnu.org/onlinedocs/libstdc++/libstdc++-html-USERS-4.3/a01696.html.
+ *
+ * Returns pointer to a temporary buffer storing the demangled function name:
+ * the caller is responsible for making a copy of it.
+ */
+const char *
+cxx_abi_demangle(const char *mangled_name);
+
+/*
+ * Free resources used by C++ ABI.
+ */
+void
+cxx_abi_cleanup(void *unused);
+#ifdef __cplusplus
+} /* extern "C" */
+#endif /* __cplusplus */
+#endif /* ENABLE_BACKTRACE */
```

Comment:
This won't compile if `ENABLE_BACKTRACE` is unset because of unterminated `{`.

Reply:
Fixed: move the `if ENABLE_BACKTRACE` out of the `ifdef __cplusplus_`.

Reply:
Sorry for this nuisance.